src/app/cube/details/components/academic-level-card/academic-level-card.component.ts

```diff
@@ -0,0 +1,18 @@
+import { Component, OnInit, Input } from '@angular/core';
+
+@Component({
+  selector: 'clark-academic-level-card',
+  templateUrl: './academic-level-card.component.html',
+  styleUrls: ['./academic-level-card.component.scss']
+})
+
+export class AcademicLevelCardComponent implements OnInit {
+  @Input() category: string;
+  @Input() academicLevels: { [name: string]: boolean };
+
+  constructor() {}
```

Comment:
Why you gotta put all these unused lifecycle hooks and constructors in stuff?

![image](https://user-images.githubusercontent.com/7431390/71187093-64124800-224c-11ea-8c52-d65bd6e2603c.png)